tests/test_data.sql

```diff
@@ -75,3 +78,36 @@ INSERT INTO sda.checksums(
         'SHA256',
         'ARCHIVED'
     );
+
+
+INSERT INTO sda.users(
+        user_name
+    ) VALUES (
+        'dummy@example.com'
+    );
+INSERT INTO sda.users(
+        user_name
+    ) VALUES (
+        'dummy2@example.com'
+    );
+
+INSERT INTO sda.users(
+        user_name
+    ) VALUES (
+        'dummy3@example.com'
+    );
+
+INSERT INTO sda.public_keys(
+        c4gh_pub_key,
+        uploaded_by
+    ) VALUES (
+        '-----BEGIN CRYPT4GH PUBLIC KEY-----
+avFAerx0ZWuJE6fTI8S/0wv3yMo1n3SuNTV6zvKdxQc=
+-----END CRYPT4GH PUBLIC KEY-----',
+        'dummy@example.com'
+    ), (
+        '-----BEGIN CRYPT4GH PUBLIC KEY-----
+wv3yMo1n3SuavFAerx0/0NTV6zvKdZWuJE6fTI8SxQc=
+-----END CRYPT4GH PUBLIC KEY-----',
+        'dummy3@example.com'
+    );
```

Comment:
```suggestion
INSERT INTO sda.users(user_name)
        VALUES ('dummy@example.com'),
                ('dummy2@example.com'), 
                ('dummy3@example.com');

INSERT INTO sda.public_keys(c4gh_pub_key,uploaded_by) 
        VALUES ('-----BEGIN CRYPT4GH PUBLIC KEY-----
avFAerx0ZWuJE6fTI8S/0wv3yMo1n3SuNTV6zvKdxQc=
-----END CRYPT4GH PUBLIC KEY-----',  'dummy@example.com'), 
('-----BEGIN CRYPT4GH PUBLIC KEY-----
wv3yMo1n3SuavFAerx0/0NTV6zvKdZWuJE6fTI8SxQc=
-----END CRYPT4GH PUBLIC KEY-----', 'dummy3@example.com');
```


Take care when you split commands over multiple lines, sometimes it makes it harder to actually see what's going on.

Reply:
good point, done.